www/footer.php

```diff
@@ -185,8 +187,6 @@
 
     //workerLog('-- footer.php');
     $return = session_write_close();
-    //workerLog('session_write_close=' . (($return) ? 'TRUE' : 'FALSE'));
+	//workerLog('session_write_close=' . (($return) ? 'TRUE' : 'FALSE'));
+	echo "</body></html>";
```

Comment:
The opening of the html and body is done in the header.php. By moving the closing of the html and body from html to php, the source document can be parsed again by regular minifiers and validators .
Change nothing to the functionality.